### ISSUE TITLE
Get this on F-Droid

### ISSUE THREAD

**Issue by Shookbelf** [whyorean/Galaxy#14]
Hi,

It would be nice to get regular updates through F-Droid. 
It will also help when Google inevitably changes their API.

**Comment by Lionirdeadman** [comment]
Yes, it will be good to have this on F-droid and Whyorean is planning to release it there.

(See here : https://gitlab.com/fdroid/fdroiddata/merge_requests/3012)

I don't know how being on F-droid could help when Google changes their API though, can you elaborate?

**Comment by Shookbelf** [comment]
> I don't know how being on F-droid could help when Google changes their API though, can you elaborate?

Just because users will get updates easier and faster, that's all.

**Comment by nikolowry** [comment]
Off-topic/Feature Request: F-Droid has been horribly unreliable the past 6+ months -- an alternative frontend to it would be awesome!

**Comment by whyorean** [comment]
@Shookbelf 
@ghost

I want the same, will publish it on Fdroid once its stable enough

@nikolowry 
I wont comment over that, but thing are about to change, maybe better..

**Comment by lionirdeadman** [comment]
F-droid has been pretty stable and it's gotten a lot better in the last month or two although they had issues back in December or so.

You should make a different issue if you want to continue discussion about this possibility though, @nikolowry

**Comment by nikolowry** [comment]
I was just throwing it out there @lionirdeadman so maintainers and contributors could see there's user interest, not going to open an issue for a trivial request.

And you're lucky you aren't experiencing this issue https://gitlab.com/fdroid/fdroidclient/issues/1271.

Apologizes for derailing this issue -- +1 to getting this on F-droid (even w my gripes)
